python/graphstorm/config/argument.py

```diff
@@ -2016,6 +2040,17 @@ def _add_link_prediction_args(parser):
             "The corresponding feature name is <feat_name>"
             "2)'--lp-edge-weight-for-loss query,adds,asin:weight0 query,clicks,asin:weight1 ..."
             "Different edge types have different weight fields.")
+    group.add_argument("--lp-major-eval-etype", type=str, default=argparse.SUPPRESS,
+            help="Canonical edge type used for selecting best model during "
+                 "link prediction training when --lp-major-eval-etype is set, "
+                 "GraphStorm will output per edge type link prediction "
+                 "evaluation metric(s) instead of averaged metric(s). And "
+                 "the metric corresponding to lp_major_eval_etype will be used "
+                 "to choose the best model. It can be in following format:"
+                "1) '--lp-major-eval-etype ALL': Use the average of the evaluation "
```

Comment:
is this the default behavior? if so, do we want users to specify all edge types in this way?